### ISSUE TITLE
Search submit button: add discernible text - A11y

### ISSUE THREAD

**Issue by lhenze** [nyudlts/ultraviolet#118]
Buttons must have discernible text. 

From Deque Systems -  Fix at least one (1) of these issues:
Element does not have inner text that is visible to screen readers

aria-label attribute does not exist or is empty

aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty

Element has no title attribute

Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by lhenze** [comment]
Note, this should be changed in the parent project -- invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header_frontpage.html

**Comment by lhenze** [comment]
I made this PR 
https://github.com/inveniosoftware/invenio-app-rdm/pull/1118

**Comment by lhenze** [comment]
This was merged -- https://github.com/inveniosoftware/invenio-app-rdm/pull/1118
